#! /bin/sh /usr/share/dpatch/dpatch-run
## chanzap_disable_r2.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: disable the nunfunctional support for R2 in chan_zap
## -- applied upstream for trunk (1.4.x)

@DPATCH@
Index: asterisk-1.2.23~dfsg/channels/Makefile
===================================================================
--- asterisk-1.2.23~dfsg.orig/channels/Makefile	2007-08-06 12:01:28.000000000 +0200
+++ asterisk-1.2.23~dfsg/channels/Makefile	2007-08-06 12:02:05.000000000 +0200
@@ -107,11 +107,6 @@
   ZAPPRI=-lpri
 endif
 
-ifneq ($(wildcard $(CROSS_COMPILE_TARGET)/usr/lib/libmfcr2.so.1)$(wildcard $(CROSS_COMPILE_TARGET)/usr/local/lib/libmfcr2.so.1),)
-  CFLAGS+=-DZAPATA_R2
-  ZAPR2=-lmfcr2
-endif
-
 ALSA_SRC=chan_alsa.c
 
 ifneq ($(wildcard alsa-monitor.h),)
